scripts/classification/train_classification.py

```diff
@@ -92,13 +94,27 @@ def parse_args():
                         help='the path to training dataset')
     parser.add_argument('--warmup_ratio', type=float, default=0.1,
                         help='Ratio of warmup steps in the learning rate scheduler.')
+    parser.add_argument('--method', type=str, default='full', choices=['full', 'bias', 'subbias', 'adapter'],
+                        help='different finetune method')

```

Comment:
Would you like to edit the README file to include results for (at least some of) the different choices (and references to the papers)?